### ISSUE TITLE
PanicHookInfo during reading logs

### ISSUE THREAD

**Issue by ppkozlowski** [aptakube/aptakube#318]
### Current and expected behavior

Hi!

I have an issue during reading logs from pods/deployment. The Aptakube application get unexpected termination during showing logs.

Steps to replicate:
1. Choose cluster: AWS EKS in my case
2. Select deployment
3. Select pod
4. Click on logs tab
5. Wait a few seconds then app is terminated

Logs:
```
[2024-10-24][10:40:33][[33mWARN[0m][tao::platform_impl::platform::event_loop::runner] NewEvents emitted without explicit RedrawEventsCleared
[2024-10-24][10:40:33][[33mWARN[0m][tao::platform_impl::platform::event_loop::runner] RedrawEventsCleared emitted without explicit MainEventsCleared
[2024-10-24][10:40:55][[33mWARN[0m][tao::platform_impl::platform::event_loop::runner] NewEvents emitted without explicit RedrawEventsCleared
[2024-10-24][10:40:55][[33mWARN[0m][tao::platform_impl::platform::event_loop::runner] RedrawEventsCleared emitted without explicit MainEventsCleared
[2024-10-24][10:41:02][[31mERROR[0m][aptakube] Panicked: PanicHookInfo { payload: Any { .. }, location: Location { file: "src\\k8s\\log_stream.rs", line: 36, col: 55 }, can_unwind: true, force_no_backtrace: false }
[2024-10-24][10:41:02][[31mERROR[0m][aptakube] Panicked: PanicHookInfo { payload: Any { .. }, location: Location { file: "src\\k8s\\log_stream.rs", line: 36, col: 55 }, can_unwind: true, force_no_backtrace: false }

```

### Environment & Operating System

Aptakube: v1.7.13
OS Name	Microsoft Windows 11 Pro
Version	10.0.22631 Build 22631

**Comment by goenning** [comment]
I was not able to reproduce it, but I'm pretty sure I found what's causing this crash.

Would you mind trying this pre-release?

https://releases.aptakube.com/Aptakube_1.7.14-beta1_universal.dmg

**Comment by goenning** [comment]
Oh, you're on windows. I'll publish a windows beta build soon

**Comment by goenning** [comment]
Would you mind trying this pre-release?

https://releases.aptakube.com/Aptakube_1.7.14_x64_en-US.msi

**Comment by ppkozlowski** [comment]
Hi @goenning 

I've installed and tested. So far, so good. I will test release after weekend during job. 

I back to give feedback.

Thanks

**Comment by goenning** [comment]
Fixed on 1.7.14, but please reopen if the issue persists!
